scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -107,36 +107,8 @@ export function ChangeImageController($scope, gettext, notify, modal, _, api, $r
     * modified crop information, point of interest and metadata changes.
     */
     $scope.done = function() {
```

Comment:
@tomaskikutis Why these check are removed? This is going break our workflow.

Checks have been added so that user don't just update the metadata and publish the item. Without these checks the users get the impression that they have to just update the metadata tab and publish the item and this caused wrong crops going out to subscriber. These checks at least ensures that users have to "click" on the crops tab and see the crops and save the item.

Reply:
Can't access the discussion. 

Configuration is fine for us.